src/posv_mixed_gmres.cc

```diff
@@ -252,20 +261,25 @@ void posv_mixed_gmres(
 
             // Wj1 = M^-1 A Vj
             slate::copy(Vj, X_lo, opts);
+            Timer t_potrs_lo_2;
             potrs(A_lo, X_lo, opts);
+            timers[ "posv_mixed_gmres::potrs_lo" ] += t_potrs_lo_2.stop();
             slate::copy(X_lo, Wj1, opts);
 
+            Timer t_hemm_lo_2;
             hemm<scalar_hi>(
                 Side::Left,
                 scalar_hi(1.0), A,
                                 Wj1,
                 scalar_hi(0.0), Vj1,
                 opts);
+            timers[ "posv_mixed_gmres::hemm_lo" ] += t_hemm_lo_2.stop();
```

Comment:
Yes, `hi`. During our original conversation I thought the last two routines inside the last `if` check were the only `hi` routines, but looking back I must of misunderstood. Since the template datatype is `scalar_hi`. Making changes.